src/compas_model/models/model.py

```diff
@@ -319,6 +320,23 @@ def add_element(
 
         return element_node
 
+    def add_group(self, name: str = None) -> ElementNode:
+        """Add a group to the model.
+
+        Parameters
+        ----------
+        name : str
+            The name of the group.
+
+        Returns
+        -------
+        :class:`ElementNode`
+            The tree node containing the group in the hierarchy.
+
+        """
+        group = Group(name=name)
+        return self.add_element(group)
```

Comment:
I'm not sure yet if we should just let the result be `ElementNode` or create another thin wrapper `GroupNode`.

Reply:
we return element instead of node (update add_element)